docs/contribute/code_review.md

```diff
@@ -38,6 +39,7 @@ APIs:
 *  CSS classes
 *  DOM structure
 *  Utility JavaScript Functions
+*  TypeScript Declaration Files
```

Comment:
Why was this added to this list? Everything else in this list is manually engineered; aren't d.ts automatically generated?

Reply:
true - removing!